src/routes/privateRpc.ts

```diff
@@ -0,0 +1,56 @@
+import { Code, ConnectError, ConnectRouter } from '@connectrpc/connect';
+import { TypeOrmError } from '../errors';
+import { ArticlePost } from '../entity';
+import { generateShortId } from '../ids';
+import createOrGetConnection from '../db';
+import { isValidHttpUrl, standardizeURL } from '../common/links';
+import { baseRpcContext } from '../common/connectRpc';
+import { PostService } from '@dailydotdev/schema';
+
+export default function (router: ConnectRouter) {
+  router.rpc(PostService, PostService.methods.create, async (req, context) => {
```

Comment:
Example:

```curl
curl --location 'POST /api/p/rpc/dailydotdev.api.posts.PostService/Create' \
--header 'Content-Type: application/json' \
--data '{
    "source_id": "unknown",
    "url": "https://example.com/url"
}'
```

It should be also callable from schema SDKs from https://github.com/dailydotdev/schema-go/blob/main/daily-api/posts/posts_grpc.pb.go and other SDKs.